internal/schema/0.12/data_block.go

```diff
@@ -43,7 +42,8 @@ func datasourceBlockSchema(v *version.Version) *schema.BlockSchema {
 			"Terraform module, but has no significance outside of the scope of a module."),
 		Body: &schema.BodySchema{
 			Extensions: &schema.BodyExtensions{
-				Count: true,
+				Count:   true,
+				ForEach: true,
```

Comment:
```suggestion
				ForEach: true, // for_each was introduced in 0.12.6, but for simplicity we report it for all 0.12+
```
Just a note to avoid any confusion of future readers.